public/common/components/match-renderer/views/index.coffee

```diff
@@ -13,24 +12,24 @@ MatchRendererView = (vm) ->
 		nxt.Class 'match-renderer'
 		nxt.Attr 'theme', 'default'
 		nxt.Binding vm.tags, (tags = []) ->
-			nxt.Fragment tags.map (tag) ->
-				{ node } = tag
-				if node.type is PuzzleTag.CLOSING
+			nxt.Fragment tags.map (item) ->
+				{tag} = item
```

Comment:
To differentiate between object literal shorthand and object pattern and it's traditional CoffeeScript syntax (although I write patterns containing nested objects with spaces, like literals:
```coffeescript
{ value: {index} } = data
```